app/concepts/cell/documentation/documentation.erb

```diff
@@ -1,46 +1,39 @@
 <section>
-  <div class="lg:hidden bg-bg-blue text-white fixed left-0 top-20 pr-1 py-3 rounded-r" id="sideNavShowButton" style="writing-mode: vertical-rl; text-orientation: upright;">
+  <div class="fixed left-0 py-3 pr-1 text-white rounded-r lg:hidden bg-bg-blue top-20" id="sideNavShowButton" style="writing-mode: vertical-rl; text-orientation: upright;">
     Chapters
   </div>
   <div class="lg:flex">
     <nav class="documentation-side-nav" id="sideNav">
-      <div class="lg:hidden absolute right-4 top-4 text-3xl text-white" id="sideNavHideButton">
+      <div class="absolute text-3xl text-white lg:hidden right-4 top-4" id="sideNavHideButton">
         X
       </div>
       <%= toc_left %>
     </nav>
-    <div class="lg:p-10 pl-8 pr-4 py-10 bg-light-grey grow">
-      <h1 class="lg:text-3xl text-2xl text-blue font-bold">
```

Comment:
@apotonick I also had to remove the gray area around top-level header as it was not getting aligned with right TOC when page gets scrolled. Instead, kept all background to white and added border around main content if that's fine.

https://github.com/apotonick/trailblazer-front/assets/6301525/36bfd337-3fff-4da3-9ab9-616e5fc73c2a

Reply:
Please revert to the designer's original version, the page looks very different without the gray parts.